pages/events/index.vue

```diff
@@ -39,6 +45,13 @@ import EventCard from '~/components/cards/EventCard.vue'
     return {
       title: 'Qiskit Experiments'
     }
+  },
+
+  async asyncData () {
+    const jsonEventsFile: any = await import('~/content/events/events-previews.json')
```

Comment:
Please, combine these two lines in one by extracting the `default` property here. You can leave the second annotation.

If you think the line would be doing too much, extract it to a function a try to only add type annotations to signatures.

Reply:
I think my way is easy to read, but it's true that `jsonEventsFile` doesn't represent what it is. I'm going to replace `jsonEventsFile` with `eventsModule`. I'm going to remove the `any` type too